keyboards/keycapsss/plaid_pad/rules.mk

```diff
@@ -23,4 +23,4 @@ AUDIO_ENABLE = no           # Audio output
 
 LAYOUTS = ortho_4x4
 
-DEFAULT_FOLDER = keycapsss/plaid_pad/rev1
+DEFAULT_FOLDER = keycapsss/plaid_pad/rev3
```

Comment:
Not sure I like this.
People running `qmk compile -kb keycapsss/plaid_pad -km <keymap>` will not know why their firmware no longer work.
I think this should still point to rev1 in order to not break user workflow.

Reply:
But then new Plaid-Pad owner wonder why the oled is not working.
Everything will work on rev1, except the status leds. I guess only experienced user used this feature. They know how to deal with it.

Reply:
Tbh, I don't really like the `DEFAULT_FOLDER` at all.
It masks a very important detail from the user.

While I understand your concern, from the project's perspective:
'not breaking problematic existing user workflow (outside of breaking change windows)' > 'creating new problematic user workflows'

I let other weigh in on it.

Reply:
The intent for `DEFAULT_FOLDER` is to give keyboard maintainers a way to move from a keyboard without revisions to a keyboard with revisions. It's not intended to be the equivalent of a symlink to the latest version.

Reply:
@skullydazed so you suggest to remove it completely, or leave it as REV1?

Reply:
Probably best, yeah.

Reply:
Either one is ok for the purpose of this PR. If you want to rip the bandaid off you can remove it now, or you can leave it pointed at rev1 for now and start warning your customers that `keycapsss/plaid_pad` will stop working in the future.

Reply:
I changed it back to rev1, because without a default folder qmk throws errors during the build process.
This might be confusing for newbies.